gnosis/protocol/tests/test_gnosis_protocol_api.py

```diff
@@ -76,23 +77,21 @@ def test_get_fee(self):
         self.assertGreaterEqual(self.gnosis_protocol_api.get_fee(order), 0)
 
     def test_get_trades(self):
+        mainnet_order_ui = "0x65F1206182C77A040ED41D507B59C622FA94AB5E71CCA567202CFF3909F3D5C4DBE338E45276630FD8237149DD47EE027AF26F9C619723D0"
         self.assertEqual(
-            self.gnosis_protocol_api.get_trades(
-                "0x9c79b5883b7f2bacbedef554a835fb07c21f4b1b046edf510554a6ba0444d2665ac255889882acd3da2aa939679e3f3d4ce"
-                "a221e72eb7b80"
-            ),
+            self.mainnet_gnosis_protocol_api.get_trades(order_ui=mainnet_order_ui),
```

Comment:
Why the change for goerli test to mainnet test?

Reply:
Because there's not enough liquidity on goerli and I cannot find a trade I can use